src/graphql/schema/utils.ts

```diff
@@ -56,3 +56,7 @@ export function mergeSchemas(...schemas: Schema[]): Schema {
   const typeDefs = R.flatten(subTypeDefs)
   return { resolvers, typeDefs }
 }
+
+export function isNotNil<A>(value: A | null | undefined): value is A {
```

Comment:
```suggestion
export function isDefined<A>(value?: A | null | undefined): value is A {
```
would be two suggestions that we can consider.